tests/integration/defs/accuracy/test_disaggregated_serving.py

```
@@ -1467,6 +1467,134 @@ def test_nixl_backend(self):
+            "backend": "pytorch",
+            "context_servers": {
+                "num_instances": 1
+            },
+            "generation_servers": {
+                "num_instances": 1
+            }
+        }
+        extra_env = {
+            "DYN_KVBM_CPU_CACHE_GB": "32",
+            "DYN_KVBM_TRTLLM_ZMQ_PORT": "20081",
+            "DYN_KVBM_LEADER_ZMQ_HOST": "0.0.0.0",
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)